source/client/game.html

```diff
@@ -15,7 +15,7 @@
   </head>
 
   <body>
-    <header>
+    <header class="icon-header">
```

Comment:
Can you add a z-index to the info button; it's currently being slightly blocked.
<img width="620" alt="Screenshot 2024-06-04 at 12 06 47 PM" src="https://github.com/cse112-sp24-group12/cse112-sp24-group12/assets/25678733/23d4d814-5016-4b15-8635-11940f468a2a">

Reply:
Done, I gave it z-index of 25 because the header has 20.